README.md

```diff
@@ -1,2 +1,3 @@
 Git practise
+more details added
```

Comment:
The nice thing about these PRs on GitHub is also that we can comment on a specific line of the code, like this.